chatbot-ui/src/components/ChatInterface.css

```diff
@@ -0,0 +1,121 @@
+.chat-container {
```

Comment:
Center in the middle, make look a bit nicer, the github link does not work. 

Also it would be great if the header is similar like the documentation.